### PR TITLE
HomePage&UploadModal v1

### PR DESCRIPTION
**Pull request by noemi-i**

### CODE REVIEW

**Comment by nMedic246** [comment]
Looks great! I'd suggest to move the icons folder to the public root folder.

File: src/components/UploadModal/UploadModal.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const dataTestIds = useMemo(
+    () => ({
+      root: dataTestId,
+    }),
+    [dataTestId],
+  );
+
+  const onUpload = () => {};
+
+  const linlkUpload = (e: React.KeyboardEvent) => {
```
Comment (nMedic246) [inline src/components/UploadModal/UploadModal.tsx]: is 'linlkUpload' a typo, or is it so on purpose?

Comment (noemi-i) [reply src/components/UploadModal/UploadModal.tsx]: Yep, it's a typo, I didn't notice that, I'll correct it

**Comment by noemi-i** [comment]
> Looks great! I'd suggest to move the icons folder to the public root folder.

I've imported icon svgs as ReactComponent, I'm not sure if it's possible from public folder, I'll do some search about it

**Comment by nMedic246** [comment]
> I've imported icon svgs as ReactComponent, I'm not sure if it's possible from public folder, I'll do some search about it

Ah yeah, sorry i missed that. Maybe it's ok to leave it like this then, idk,  maybe Ivan has some more knowledge of that.

**Review by vlahovivan — CHANGES_REQUESTED** [review]
Looks good, there is only a few minor things I've noticed:
- the "or" part is missing in the modal
- left padding in the input box could be increased a bit
  - just add `padding-left: 8px` to `.modal-link-section`
- you can also change the color of input text in `.modal-link-section`, as well as the placeholder
```css
color: var(--sa-neutral-color-5);

&::placeholder {
  color: var(--sa-neutral-color-3);
}
```

After fixing that, and the things that Nika noticed, I think we can merge this and then add the Button component instead of custom buttons once we merge my PR as well.
The other option is to first merge my PR, then add the buttons here, and merge this PR.

**Review by nMedic246 — APPROVED** [review]
I think it's all good now.

**Review by vlahovivan — APPROVED** [review]

**Pull request merged**